src/components/Board/Board.container.js

```diff
@@ -1517,10 +1525,11 @@ export class BoardContainer extends Component {
     //return condition
     newBoard.tiles.forEach(async tile => {
       if (tile && tile.loadBoard && !tile.linkedBoard) {
+        if (folderTile.loadBoard === firstPastedFolderId) return;
```

Comment:
This condition will be always true such as the params `folderTile.loadBoard` and `firstPastedFolderId` are equals on the `pasteBoardsRecursively` function call.